### PR TITLE
Add user authentication

### LINKED ISSUE #10: Add user authentication
This should work similarly to Nook/Nook Watcher

### PR DESCRIPTION
**Pull request by ktkization**
closes #10

### CODE REVIEW

File: webapp/lib/controller.dart [hunk 1]
```diff
@@ -48,10 +66,12 @@ void init() async {
 void initUI() {
   window.location.hash = '#/dashboard'; //TODO This is just temporary initialization becuase we don't have a complete app
   router.routeTo(window.location.hash);
+  view.navView.projectTitle = 'COVID IMAQAL Batch replies (Week 12)'; //TODO To be replaced by data from model
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: This is the package title not the project title. I think here it should read just 'COVID IMAQAL', and then in `loadConfigurationView` the title should be rewritten as 'COVID IMAQAL Batch replies (Week 12)' (with the todo to be replaced with data from the model)

File: webapp/lib/view.dart [hunk 2]
```diff
@@ -59,6 +119,38 @@ class ContentView {
   }
 }
 
+class AuthMainView {
+  DivElement authElement;
+  ButtonElement _signInButton;
+
+  final descriptionText1 = 'Sign in to Nook Watcher where you can monitor Nook deployments.';
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]:
```suggestion
  final descriptionText1 = 'Sign in to Katikati';
```

**Review by marianamarasoiu — APPROVED** [review]
Thanks @ktkization just a couple of comments, but LGTM!

I also noticed a couple of things behaving oddly with the routing while testing this PR, so I've opened a couple of issues (#12 and #13) - but I think best to handle them in separate PRs rather than here

File: webapp/lib/controller.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: CHanged it. I'll do the rewriting when I address issues raised in the review of #15 so as to reduce the number of merge conflicts

File: webapp/lib/view.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Yes. That's right. Thanks

**Review by marianamarasoiu — APPROVED** [review]
Thanks! LGTM and merging...

**Pull request merged**